python/version_details/shotgun_entities/card_delegate.py

```diff
@@ -167,11 +171,8 @@ def _get_painter_widget(self, model_index, parent):
         if model_index in self._widget_cache:
             widget = self._widget_cache[model_index]
 
-            if self.fields != widget.fields:
-                widget.fields = self.fields
-                self.sizeHintChanged.emit(model_index)
-
-            return widget
+            if sorted(self.fields) == sorted(widget.fields):
```

Comment:
Not a big deal, but if we had made fields a set during __init__, then we wouldn't have to sort these every single time. If fields is read-only, then frozenset would make even more sense.